adr/0007-integration-config-yaml-structure.md

```diff
@@ -47,6 +47,10 @@ sensor:
     username: user
 ```
 
+### Exceptions
+
+- Implementing an optional `unique_id` configuration key to set the unique ID of the entity created by the platform is allowed.
```

Comment:
Should we say something about what platforms are allowed to do this. It's not allowed for all platforms, only platforms that create entities that derive their state from other integrations or platforms for transport implementing integrations, or if we have made some special exception.

Reply:
Maybe say: May not conflict with ADR-0010?

Reply:
It has been added to `command_line` that is not excepted from ADR0010 or derived from other entities or is a transport protocol, so changing the wording here does not reflect what has already been accepted.

Reply:
The command line defines its own "protocol". As in, it does not connect to a device or service in a normal sense. It is safe to say it doesn't fall under adr-0010 (at least not intentional)